### PR TITLE
feature/spreadsheet-tests

### PR DESCRIPTION
**Pull request by s-block**

### CODE REVIEW

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

File: cla_public/apps/checker/tests/test_spreadsheet.py [hunk 1]
```diff
@@ -0,0 +1,172 @@
...
+        except (ValueError):
+            return False
+
+    def number_if_yes(self, n, d):
+        return int(n) if n and int(n) > 0 and d == YES else None
+
+    def get_total_if_partner(self, your_value, partner_value):
+        val = 0.00
+        if your_value:
+            val += float(your_value)
```
Comment (andyhd) [inline cla_public/apps/checker/tests/test_spreadsheet.py]: I don't think there should be any floats used here. Money values should be integers in pence.

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

**Comment by s-block** [comment]
retest this please

**Pull request merged**